src/context/AuthProvider.tsx

```diff
@@ -0,0 +1,6 @@
+"use client";
+
+import { SessionProvider } from "next-auth/react";
+export default function App({ children }: { children: React.ReactNode }) {
```

Comment:
whats the point of this element, could you not wrap all the elements in layout with sessionprovider

Reply:
I had doubts regarding that, so i checked on the tutorial how it was done. I'll update it